types/wicket/index.d.ts

```diff
@@ -1,46 +1,48 @@
-declare var Wkt: WktModule.WktStatic;
+export {};
 
-declare namespace WktModule {
-    interface WktStatic {
-        new(obj?: any): Wkt;
-        beginsWith(str: string, sub: string): boolean;
-        endsWith(str: string, sub: string): boolean;
-        delimiter: string;
-        isArray(obj: any): boolean;
-        trim(str: string, sub: string): string;
-        Wkt: typeof Wkt;
-    }
+declare class WktClass {
+    constructor();
+    delimiter: string;
+    wrapVertices: string;
+    regExes: string;
+    components: string;
+    isCollection(): boolean;
+    sameCoords(a: any, b: any): boolean;
+    fromObject(obj: any): WktClass;
+    toObject(config?: any): any;
+    toString(config?: any): string;
+    fromJson(obj: any): WktClass;
+    toJson(): any;
+    merge(wkt: string): WktClass;
+    read(str: string): WktClass;
+    write(components?: any[]): string;
+    type: string;
+    extract: Extract;
+    isRectangle: boolean;
+}
 
-    class Wkt {
-        constructor();
-        delimiter: string;
-        wrapVertices: string;
-        regExes: string;
-        components: string;
-        isCollection(): boolean;
-        sameCoords(a: any, b: any): boolean;
-        fromObject(obj: any): Wkt;
-        toObject(config?: any): any;
-        toString(config?: any): string;
-        fromJson(obj: any): Wkt;
-        toJson(): any;
-        merge(wkt: string): Wkt;
-        read(str: string): Wkt;
-        write(components?: any[]): string;
-        type: string;
-        extract: Extract;
-        isRectangle: boolean;
-    }
+export function beginsWith(str: string, sub: string): boolean;
+export function endsWith(str: string, sub: string): boolean;
+export let delimiter: string;
+export function isArray(obj: any): boolean;
+export function trim(str: string, sub: string): string;
+export let Wkt: typeof WktClass;
```

Comment:
As I mentioned above, this should really just be combined with `WktClass` as `export class Wkt { ... }`